.github/workflows/main.yaml

```diff
@@ -23,8 +23,8 @@ jobs:
       - name: Install
         run: |
           git config --global --add safe.directory /__w/cmip6-downscaling/cmip6-downscaling
-          python -m pip install --no-deps -e .
-          python -m pip install -U intake intake-esm cf-xarray
+          python -m pip install -e .
+          python -m pip install -U cf-xarray
```

Comment:
do the tests fail with the version of cf-xarray included in the docker images? if so, please specify which version is required in requirements.txt rather than upgrading in the CI, similar to the steps taken for intake-esm.

Reply:
Good call @maxrjones. Looks the the cf-xarray 0.7.2 was causing the test failures.